packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -15,6 +16,7 @@ contract DeployConfig is Script {
     string internal _json;
 
     address public finalSystemOwner;
+    // todo(maurelian): rename as this has been moved to the SuperchainConfig
```

Comment:
its hard to tell which todos you want to resolve now and in the future

Reply:
Yeah, that's fair, I've just removed them for now.